src/Access/RoleCache.h

```diff
@@ -16,7 +16,7 @@ using RolePtr = std::shared_ptr<const Role>;
 class RoleCache
 {
 public:
-    explicit RoleCache(const AccessControl & access_control_);
+    explicit RoleCache(const AccessControl & access_control_, int expiration_time);
```

Comment:
Let's add something to the name which will say that `expiration_time` in seconds.